mne/forward/_field_interpolation.py

```diff
@@ -420,3 +421,88 @@ def make_field_map(evoked, trans='auto', subject=None, subjects_dir=None,
         surf_maps.append(this_map)
 
     return surf_maps
+
+
+def _transform_instance(inst_from, info_to):
+    """ Map the inst_from to the head position in info_to"""
+    info_from = inst_from.info
+    picks_from = pick_types(info_from, meg=True, eeg=False, ref_meg=False,
+                           exclude='bads')
+
+    # The transformation to be done to go to the new head position
+    from ..transforms import invert_transform, apply_trans
+    trans = np.dot(info_to['dev_head_t']['trans'],
+                   invert_transform(info_from['dev_head_t'])['trans'])
+    chs_pos_from = inst_from._get_channel_positions(picks_from)
+    pos = chs_pos_from[:, :3]
+    vec_x = chs_pos_from[:, 3:6]
+    vec_y = chs_pos_from[:, 6:9]
+    vec_z = chs_pos_from[:, 9:]
+    chs_pos_from = list()
+    for pts in [pos, vec_x, vec_y, vec_z]:
+        chs_pos_from.append(apply_trans(trans, pts))
```

Comment:
looks like you can save some lines of code here using tiling / reshaping. Not sure ...